app/models/fandom.rb

```diff
@@ -14,9 +14,8 @@ class Fandom < Tag
 
   scope :by_media, lambda {|media| where(media_id: media.id)}
 
-  def self.unwrangled
-    joins(:common_taggings).
-    where("unwrangleable = 0 AND common_taggings.filterable_id = ? AND common_taggings.filterable_type = 'Tag'", Media.uncategorized.try(:id))
+  def unwrangled?
+    Media.uncategorized.fandoms.include?(self)
```

Comment:
Would `include?` eager-load all the fandoms under uncategorized? Should we use [`exists?`](https://api.rubyonrails.org/v6.1/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many) instead?

Reply:
Rails seems to be smartly converting the `include?` to `exists?` on its own there:
```
irb(main):003:0> Media.uncategorized.fandoms.include?(test)
  Tag Load (0.4ms)  SELECT `tags`.* FROM `tags` WHERE (tags.name = 'Uncategorized Fandoms') ORDER BY `tags`.`id` ASC LIMIT 1
Query Trace:
      app/models/tag.rb:575:in `find_by_name'
      app/models/tag.rb:582:in `find_or_create_by_name'
      app/models/media.rb:13:in `uncategorized'
  Tag Exists? (0.2ms)  SELECT 1 AS one FROM `tags` INNER JOIN `common_taggings` ON `tags`.`id` = `common_taggings`.`common_tag_id` WHERE `common_taggings`.`filterable_id` = 1063753922 AND `common_taggings`.`filterable_type` = 'Tag' AND `tags`.`type` = 'Fandom' AND `tags`.`id` = 130714128 LIMIT 1
```